coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -57,11 +57,31 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         filename = source_range.file
 
         ignore_diff = Diff(original_file_dict[filename])
-        ignore_diff.change_line(
-            source_range.start.line,
-            original_file_dict[filename][source_range.start.line-1],
-            original_file_dict[filename][source_range.start.line-1].rstrip() +
-            '  ' + ignore_comment)
+        """
+        Apply the ignore comment using 'and' with the previous ignore,
+        if the same line already has a previous ignore comment.
+        """
```

Comment:
If this is a normal comment, rather use `#` instead of `"""`, as those are actually strings in Python.

Reply:
@Makman2 this is a docstring specifying what the next lines of code will do.